src/components/About/About.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { StyleSheet, Text, View, ImageBackground } from 'react-native';
+import { StyleSheet, Text, View } from 'react-native';
```

Comment:
About/About.js: Code is clean and 'ImageBackground' is removed since it is not used here. `fontWeight: 'lighter',` is still not incorporated.